integration/sql/tests/test_utils/mod.rs

```diff
@@ -0,0 +1,17 @@
+use openlineage_sql::{parse_sql, DbTableMeta, SqlMeta};
+use sqlparser::dialect::PostgreSqlDialect;
+
+pub fn test_sql(sql: &str) -> SqlMeta {
+    parse_sql(sql, Box::new(PostgreSqlDialect {}), None).unwrap()
```

Comment:
I will try to avoid `unwrap` in code. 
This article has great alternatives https://dmerej.info/blog/post/killing-unwrap/

Reply:
Fixed.